### PR TITLE
Update OpsGenie to include who is on call

### PR DESCRIPTION
**Pull request by PeyGis**
# Description

This PR adds a section that points users on how to ingest OpsGenie's on-call data into Port

## Updated docs pages

https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/webhook/examples/opsgenie

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/opsgenie.md [hunk 1]
```diff
@@ -53,3 +53,8 @@ Create the following blueprint definition and webhook configuration:
 7. Click **Save integration**
 
 Done! any change that happens to an OpsGenie alert (created, acknowledged, etc.) will trigger a webhook event that OpsGenie will send to the webhook URL provided by Port. Port will parse the events according to the mapping and update the catalog entities accordingly.
+
+## Ingest who is on-call
+In this example you will create a blueprint for `service` entity that ingests who is on call data from OpsGenie using REST API. Then you will add some Python code to create new entities in Port every time a Github Action or GitLab Pipeline is triggered by a schedule. The links below guide you on how to accomplish this task on your preferred environment.
+[Github Action](https://github.com/PeyGis/OpsGenie)
```
Comment (danielsinai) [inline docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/opsgenie.md:59]: Please change it to GitHub Workflow

Comment (danielsinai) [inline docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/opsgenie.md:59]: Please move those repos to port-labs both in github and gitlab

Comment (danielsinai) [reply docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/opsgenie.md:59]: Ill send you invites right away

**Review by danielsinai — CHANGES_REQUESTED** [review]
Great work buddy 🦢

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/opsgenie.md [hunk 2]
```diff
@@ -53,3 +53,9 @@ Create the following blueprint definition and webhook configuration:
 7. Click **Save integration**
 
 Done! any change that happens to an OpsGenie alert (created, acknowledged, etc.) will trigger a webhook event that OpsGenie will send to the webhook URL provided by Port. Port will parse the events according to the mapping and update the catalog entities accordingly.
+
+## Ingest who is on-call
+In this example we will create a blueprint for `service` entities with a `on-call` property that will be ingested directly from OpsGenie.
```
Comment (Avivlavie) [inline docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/opsgenie.md:58]:
```suggestion
In this example we will create a blueprint for `service` entities with an `on-call` property that will be ingested directly from OpsGenie. 
```

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/opsgenie.md [hunk 3]
```diff
@@ -53,3 +53,9 @@ Create the following blueprint definition and webhook configuration:
 7. Click **Save integration**
 
 Done! any change that happens to an OpsGenie alert (created, acknowledged, etc.) will trigger a webhook event that OpsGenie will send to the webhook URL provided by Port. Port will parse the events according to the mapping and update the catalog entities accordingly.
+
+## Ingest who is on-call
+In this example we will create a blueprint for `service` entities with a `on-call` property that will be ingested directly from OpsGenie. 
+The exmaples below are pulling data from the OpsGenie REST Api, in a defined scheduled period using GitLab Pipelines or GitHub Workflows, and reports the data to Port as a property to the `service` blueprint.
```
Comment (Avivlavie) [inline docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/opsgenie.md:59]:
```suggestion
The examples below pull data from the OpsGenie REST Api, in a defined scheduled period using GitLab Pipelines or GitHub Workflows, and report the data to Port as a property to the `service` blueprint.
```

**Review by Avivlavie — APPROVED** [review]

**Pull request merged**